.github/workflows/tests.yml

```diff
@@ -364,12 +366,15 @@ jobs:
     strategy:
       matrix:
         include:
+          - name: test default formula (Ubuntu 24.04)
+            runs-on: ubuntu-latest
+            container: ghcr.io/homebrew/ubuntu24.04:latest
           - name: test default formula (Ubuntu 22.04)
             runs-on: ubuntu-latest
             container: ghcr.io/homebrew/ubuntu22.04:master
```

Comment:
Should this line also be `ghcr.io/homebrew/ubuntu22.04:latest`?

Reply:
`master` is a little better as it's quicker to react to Dockerfile regressions but not significant enough to start pushing `master` tags for other versions.

So really it doesn't matter much

Reply:
We use `master` for this since it pulls the latest changes from HEAD, rather than using the latest tag.